### ISSUE TITLE
Timer state is reset on app leave and close

### ISSUE THREAD

**Issue by moWerk** [AsteroidOS/asteroid-timer#5]
Expected behaviour is that set timers do run until end (alert) or user reset not until app leave or close afaiu. 
Using nightly build 20180115 on dory and lenok.

**Comment by FlorentRevest** [comment]
If we want to keep counting while asteroid-timer is closed, we need to have some kind of daemon running in background. I detailed in [this post](https://github.com/AsteroidOS/asteroid-timer/issues/6#issuecomment-358376370) a couple of possible solutions using timed, systemd, a custom standalone daemon or a custom daemon coupled with timed or systemd. Again, there is no magic solution and they all get very complex for a simple task.

For now (for the time of v1.0), this is just the accepted behavior. Anyone can feel free to contribute a better solution.

**Comment by moWerk** [comment]
Thank you for taking your time and explain!
Is it even approriate to open those obvious and propably already discussed feature requests as an issue?
I am fully aware that much work is happening in areas not directly visible to users and i do not want to push you unnecessarily but for casual testers like me it is hard to keep track of what is worthy to report :D

**Comment by FlorentRevest** [comment]
No don't worry, it's far from being obvious and very badly documented (probably just a couple of messages from me grumbling on IRC a couple of months ago).

I didn't close your issues precisely because I consider those as issues and also because it was the opportunity to document "properly" the reason why it happens. It's just low priority for me right now.

Thank you for your testing and please keep reporting anything that isn't clear!
